src/AkismetField.php

```diff
@@ -81,22 +93,22 @@ public function getSpamMappedData()
      * This function first gets values from mapped fields and then check these values against
      * Mollom web service and then notify callback object with the spam checking result.
      * @param Validator $validator
-     * @return 	boolean		- true when Mollom confirms that the submission is ham (not spam)
-     *						- false when Mollom confirms that the submission is spam 
-     * 						- false when Mollom say 'unsure'. 
-     *						  In this case, 'mollom_captcha_requested' session is set to true 
-     *       				  so that Field() knows it's time to display captcha 			
+     * @return  boolean     - true when Mollom confirms that the submission is ham (not spam)
+     *                      - false when Mollom confirms that the submission is spam
```

Comment:
But... but this is Akismet, not [Mollom](https://www.mollom.com/)!

@tractorcow must have based it on the [mollom module](https://github.com/silverstripe-archive/silverstripe-mollom) - which is now correctly archived, since:

>**End-of-Life Announcement:** As of 2 April 2018, Acquia will no longer support or maintain the Mollom product. [Learn More](https://www.mollom.com/eol)